src/modules/navigator/rtl_direct.cpp

```diff
@@ -310,6 +310,7 @@ void RtlDirect::set_rtl_item()
 			dest.alt = loiter_altitude;
 
 			setMoveToPositionMissionItem(_mission_item, dest, rtl_heading_mode);
+			_navigator->reset_position_setpoint(pos_sp_triplet->previous);
```

Comment:
Why is this now required?

Reply:
@sfuhrer Because we don't want line following when moving to the land waypoint during hover.